docs/provisioning/silo_vnet_newstorage.md

```diff
@@ -0,0 +1,83 @@
+# Create a silo behind a vnet with a new private storage
+
+This tutorial applies in the case you want to create a **completely new storage** account in a federated learning silo in a given region.
```

Comment:
maybe add a sentence or 2 explaining what kind of threats this protects from?

Reply:
Let me file an issue for that and implement later, I want to gather more info about what it actually protects from and make sure I get "authorivative" quotes from our docs.

Reply:
https://github.com/Azure-Samples/azure-ml-federated-learning/issues/133